### PR TITLE
DTGB-592: Added filter nav component and styling for outdoor location…

### PR DESCRIPTION
**Pull request by meirege**
… detail pages.

## Screenshots (if appropriate):
screen shot 2018-10-19 at 09 10 10

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have updated the documentation accordingly.
- [ ] I have updated the style guide CHANGELOG accordingly.
- [ ] I have updated the gent_base CHANGELOG accordingly.
- [ ] I have ran gulp axe on the compiled files and found no errors.

### CODE REVIEW

**Review by hstandaert — APPROVED** [review]

File: styleguide/components/31-molecules/nav-filter/nav-filter.twig [hunk 1]
```diff
@@ -0,0 +1,7 @@
+<nav class="filter--nav">
+  {% include '@list' with {
```
Comment (delrueba) [inline styleguide/components/31-molecules/nav-filter/nav-filter.twig]: Missing hidden h2 and aria-labelledby to name this nav region.
See breadcrumbs, paging...

**Review by delrueba — CHANGES_REQUESTED** [review]
Not a fan of the name,
it was unclear what this component was supposed to be. 

It's a TOC right?

File: styleguide/components/31-molecules/nav-filter/nav-filter.config.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
+'use strict';
+
+module.exports = {
+  name: 'Nav filter',
+  handle: 'nav-filter',
+  status: 'alpha',
+  preview: '@preview',
+  context: {
+    list: [
+      '<a href="/ga-voor-een-onvergetelijke-schoolstage" class="active">Alle opleidingsniveaus</a>',
```
Comment (delrueba) [inline styleguide/components/31-molecules/nav-filter/nav-filter.config.js]: English please
Also, if it's TOC the links should be anchor links so screenreaders say "same page link".

**Review by delrueba — CHANGES_REQUESTED** [review]

File: styleguide/components/31-molecules/nav-filter/_nav-filter.scss [hunk 3]
```diff
@@ -0,0 +1,13 @@
+.filter--nav {
+  ul > li {
+    margin-right: 0;
+
+    + li::before {
+      @include theme('color', 'color-primary--lighten-3', 'filter-nav-longdash-color')
+
+      display: inline-block;
+      padding: 0 .2rem;
+      content: '—';
```
Comment (delrueba) [inline styleguide/components/31-molecules/nav-filter/_nav-filter.scss]: Cute smiley.

File: templates/custom/nodes/node--outdoor-location--full.html.twig [hunk 4]
```diff
@@ -1,5 +1,12 @@
 {% extends 'node--detail--layout.html.twig' %}
 
   {% block article_content_summary %}
+    <div class="filter--nav">
```
Comment (delrueba) [inline templates/custom/nodes/node--outdoor-location--full.html.twig]: Let's be semantic.
Don't forget to include the h2 etc here too.

**Review by delrueba — CHANGES_REQUESTED** [review]

**Review by delrueba — APPROVED** [review]

**Pull request merged**